bin/build_sidecar_injector.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
should this go in `sidecar_injector/` to facilitate moving the content later?

Reply:
I agree

Reply:
addressed